additional-databases/neptune/README.md

```diff
@@ -54,3 +58,24 @@ You should see a report, similar to the one below, indicating the status and det
 ```
 
 Now, you should be able to run queries from your local Jupyter graph notebook to your Neptune clusters!  When you're ready to close the connection, use Ctrl+D to exit.
+
+### Troubleshooting
+
+If you're having trouble with seeing a successful status to Neptune, try the following:
+
+<b>EC2 login: </b> make sure your key-pair (.pem) file is valid.  Use a basic form of the ssh command to see if you can login to EC2.
+`ssh -i path/to/keypairfilename.pem ec2-user@yourec2instanceendpoint` 
+
+A successful login with the above command will show a login confirmation like the one below:
+```
+Last login: Tue Dec 15 21:22:56 2020 from X.X.X.X
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux 2 AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-2/
+```
+<b>DNS flush: </b> after setting your hosts file, do a DNS flush to make sure your changes are reflected in the DNS.
```

Comment:
I have had to do this once client side. Does this actually work server (EC2) side? I have no idea as I have never needed to do it.

Reply:
Ah I will clarify, as I also did client-side.